docs/program/apis/connectivity.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Client Sessions and Robot Network Connectivity"
+linkTitle: "Network Connectivity Issues"
+weight: 20
+type: "docs"
+description: "Whan a robot loses its connection to the internet, all client sessions will timeout and end by default."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet over a LAN or WAN and can no longer communicate with clients through [the Viam app](https://app.viam.com).
+
+## How does `viam-server` work if your robot loses its network connection?
+
+When a robot loses its connection over LAN or WAN, it can no longer communicate with clients through [the Viam app](https://app.viam.com).
+`viam-server` will timeout and the `SessionsClient` will end any current client [*sessions*](/program/apis/sessions/) on this robot.
```

Comment:
```suggestion
After no heartbeat has been received over a timeout period, `viam-server` will end any current client [*sessions*](/program/apis/sessions/) on this robot.
```
To clarify, sessions are tracked in viam-server itself. When the client connection is interrupted/disconnected, the client is no longer able to to send heartbeats. The viam-server will end the session after a timeout period where no heartbeat is received, it then cancels operations that belong to that session.